modules/op-release-notes-1-10.adoc

```diff
@@ -211,4 +211,16 @@ With this update, {pipelines-title} General Availability (GA) 1.10.4 is availabl
 
 * Before this update, the `pipelinerun.timeouts` field was reset to the `timeouts.pipeline` value, ignoring the `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.
 
-* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
\ No newline at end of file
+* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
+
+[id="release-notes-1-10-5_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.5
+
+With this update, {pipelines-title} General Availability (GA) 1.10.5 is available on {product-title} 4.10 in addition to 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-10-5_{context}"]
+=== Fixed issues
```

Comment:
one more bugfix was merged in upstream pipelines - https://github.com/tektoncd/pipeline/pull/6861

```
Merge podTemplate specified in pipelineRun.spec.taskRunSpecs[].podTemplate along with pipelineRun.spec.podTemplate instead of only considering the one specified at the taskRunSpecs.
```

Reply:
we should add this as well